app/js/components/manageApps/Addon.jsx

```diff
@@ -330,7 +336,7 @@ class Addon extends Component {
           <div className="title-container">
             <h3 id="addon-name">{app.name}</h3>
             <p id="addon-description">{app.description}</p>
-            {app.uuid ?
+            {app.uid || app.uuid ?
```

Comment:
What are the cases where it is uid instead of uuid?

Reply:
The new response i am getting when i fetch modules is its uid and no longer uuid.
I left the uuid deliberately for future response

Reply:
Ignore the future. We shall change it if that future comes.

Reply:
Okay, i have updated the PR